docker-compose.yml

```diff
@@ -59,6 +59,12 @@ services:
     command: python -m data_rentgen.consumer
     restart: unless-stopped
     env_file: .env.docker
+    healthcheck:
```

Comment:
Please check documentation if consumer install example is correct. There are some highlighted lines here

Reply:
update